Makefile

```diff
@@ -42,23 +42,33 @@ ELECTRON_PATH=$(IMPORT_PATH)/outline/electron
 
```

Comment:
Perhaps we should move the entire directory to `/outline/internal/electron`, to make it clear this code is not to be used externally.

Reply:
Same for the android and apple directories.

Reply:
Agree, I will start with the electron one first.